File: 프로그래머스/lv1/12921. 소수 찾기/소수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 1;
        boolean flag = true;
        for (int i = 3; i<=n; i++){

            for (int j = 2; j<Math.sqrt(i)+1; j++){
               if (i%j == 0){
                   flag = false;
                   
                   break;
               }            
            }
            if(flag) {
                answer++;

                }
            flag = true;
        }
        return answer;
    }
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
import math

N = int(input())
numbers = [int(i) for i in input().split()]
prime = True
answer = 0
for i in numbers:
	if i == 1:
		continue
	for j in range(2, int(math.sqrt(i)) + 1):
		if i % j ==0:
			prime = False
			break
	if prime :
		answer += 1
	prime = True

print(answer)